packages/flutter/lib/src/widgets/gesture_detector.dart

```diff
@@ -124,6 +124,11 @@ class GestureRecognizerFactoryWithHandlers<T extends GestureRecognizer> extends
 ///   ),
 /// )
 /// ```
+///
+/// ## Debugging
+///
+/// To see how large the hit test box of a [GestureDetector] is for debugging
+/// purposes, set [debugPaintPointersEnabled] temporarily to true.
```

Comment:
it's unclear what "temporarily" means here -- for one frame?

Reply:
I mean temporarily in the sense of "don't ship your app with this setting". I'll remove the word. That should already be clear by "for debugging purposes".